README.md

```diff
@@ -13,7 +13,7 @@ Saga requires that babel-polyfill be installed and configured for your project:
 ```
 npm install babel-polyfill
 ```
-Then make you you import 'babel-polyfill' in your entry file (ie index.js)
+Then make you import 'babel-polyfill' in your entry file (ie index.js)
```

Comment:
looks like this wasn't a duplicate work, but rather a typo where the first you should be "sure"... ie Then make sure you...